pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -267,6 +267,11 @@ func (l *LibvirtDomainManager) startMigration(vmi *v1.VirtualMachineInstance, op
 	if err != nil {
 		log.Log.Object(vmi).Reason(err).Error("Live migration failed.")
 		l.setMigrationResult(vmi, true, fmt.Sprintf("%v", err), "")
+
+		log.Log.Object(vmi).Info("migration failed, reconnect sriov host-devices")
+		if reconnectSriovDevicesError := l.hotPlugHostDevices(vmi); err != nil {
+			log.Log.Object(vmi).Reason(reconnectSriovDevicesError).Error("failed to hot-plug host-devices")
+		}
```

Comment:
This whole error block seems redundant/no-op to me.
`asyncMigrate` will always return `nil`, there is nothing there to return an error.

Reply:
Thanks for the head-up, you right I will remove it